test/show.jl

```diff
@@ -1,19 +1,103 @@
 module TestShow
     using Base.Test, DataFrames
 
-    df = DataFrame(A = 1:3, B = ["x", "y", "z"])
+    # In the future newline characte \n should be added to this test case
+    df = DataFrame(A = 1:4, B = ["x\"", "∀ε⫺0: x+ε⫺x", "z\$", "ABC"],
+                   C = Float32[1.0, 2.0, 3.0, 4.0])
```

Comment:
Could you add a `null` value somewhere so that this is covered (unless it's done elsewhere already)?

Reply:
It is already covered I believe in line:
```
df = DataFrame(Fish = ["Suzy", "Amir"], Mass = [1.5, null])
```
at the end of the file

Reply:
Yes, but `showcols` isn't tested there. Would be worth adding a test.

Reply:
added `showcols` test